Base/Python/slicer/util.py

```diff
@@ -780,13 +780,34 @@ def loadColorTable(filename, returnNode=False):
 
 
 def loadFiberBundle(filename, returnNode=False):
-    """Load node from file.
+    """Load fiber bundle node from file.
+
+    .. warning::
+    To ensure the FiberBundleFile reader is registered, the ``SlicerDMRI``
+    extension needs to be installed.
 
     :param filename: full path of the file to load.
     :param returnNode: Deprecated.
     :return: loaded node (if multiple nodes are loaded then a list of nodes).
       If returnNode is True then a status flag and loaded node are returned.
     """
+
+    # Check if SlicerDMRI is installed
+    extension_name = "SlicerDMRI"
+    em = slicer.app.extensionsManagerModel()
+    if not em.isExtensionInstalled(extension_name):
+        errorMessage = f"{extension_name} not installed."
+        errorMessage += "\n" + f"Install the {extension_name} extension"
```

Comment:
```suggestion
```

Reply:
@jcfr Can you please agree with @jamesobutler whether this is required or not? He explicitly requested the check in https://github.com/Slicer/Slicer/pull/7375#discussion_r1451751391

Reply:
@jhlegarreta I agree with @jcfr. You should only check the status of SlicerDMRI being installed if the file reader is not detected. No need to check if SlicerDMRI is installed every time this utility method is called. Only when there will be a failure.

Reply:
I will follow up asap, and will propose an updated PR. Thanks for the patience :pray:

Reply:
Solved with the last push force. Thanks @jcfr.